src/plugins/initiatives/initiativeEntry.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import {type TimestampMs} from "../../util/timestamp";
+import {type NodeAddressT} from "../../core/graph";
+import {type InitiativeWeight} from "./initiative";
+import {type NodeEntryJson} from "./nodeEntry";
+
+/**
+ * Stored inside a initiatives.json db file with other initiatives.
+ * presently the `id` prop is used as the key inside the db file
+ */
+export type InitiativeEntry = {|
+  +id: string, // GUID
+  +title: string,
+  +timestampMs: TimestampMs,
+  +weight: InitiativeWeight,
+  +completed: boolean,
+  // should be user nodes
+  +champions: $ReadOnlyArray<NodeAddressT>,
+  // other initiative nodes
+  +dependencies: $ReadOnlyArray<NodeAddressT>,
+  // an activity node. Will be more difficult to specify this
```

Comment:
What do you have in mind when you say an activity node? why will it be more difficult to specify?

Reply:
I just see references as non-user-like nodes, and I'm not sure there's a way to do this statically. Right now I"m filtering on user-types to create an array of "activity" nodes for this field.